src/alchemlyb/workflows/abfe.py

```diff
@@ -201,6 +219,7 @@ def run(
         overlap="O_MBAR.pdf",
         breakdown=True,
         forwrev=None,
+        n_jobs=-1,
```

Comment:
see above
- Is -1 safe as new default?
- add docs (explanation)
- add versionchanged